transport.go

```diff
@@ -33,6 +36,7 @@ type Transport struct {
 
 	mu    *sync.Mutex  // mu protects token
 	token *accessToken // token is the installation's access token
+	ctx   context.Context
```

Comment:
`ctx` is not protected by `mu`, is it? Putting it here as you did makes it seem like it is.

See https://dmitri.shuralyov.com/idiomatic-go#mutex-hat.